### PR TITLE
Introduced broken template message

### LINKED ISSUE #247: Issues with creating/modifying articles that no longer have a matching template.
| Q | A
| --- | ---
| Bug? | no
| New Feature? | yes
| SuluArticleBundle Version | 0.6.2
| Sulu Version | 1.6

#### Actual Behavior

when renaming the key of a template but not the name of the file itself it's still possible to open a article in the admin. but it has multipage enabled and you won't be able to save the article.  Also it's possible to create a new article based on that template (although the page won't load)

#### Expected Behavior

Should give a clear warning that the template can not be found. and avoid opening the page in that case, to avoid issues with the content.

#### Steps to Reproduce

Rename the `<key>` in an article template without changing the filename.

#### Possible Solutions

There should be a template checker cause this could cause weird issues. To my knowledge something similar already exists in the pages section.

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #247
| Related issues/PRs | massiveart/husky#777
| License | MIT

#### What's in this PR?

This PR introduces a broken template message (like in the page tree).

#### To Do

- [x] Translations

### CODE REVIEW

File: Resources/translations/sulu/backend.nl.xlf [hunk 1]
```diff
@@ -194,6 +194,10 @@
                 <source>sulu_article.edit.page</source>
                 <target>Pagina</target>
             </trans-unit>
+            <trans-unit id="broken-template-01" resname="sulu_content.broken-template.title">
+                <source>sulu_content.broken-template.title</source>
+                <target>Le gabarit de artikel est cassé</target>
```
Comment (trickreich) [inline Resources/translations/sulu/backend.nl.xlf]: Is this translation correct?

File: Resources/translations/sulu/backend.fr.xlf [hunk 2]
```diff
@@ -194,6 +194,10 @@
                 <source>sulu_article.edit.page</source>
                 <target>Page</target>
             </trans-unit>
+            <trans-unit id="broken-template-01" resname="sulu_content.broken-template.title">
+                <source>sulu_content.broken-template.title</source>
+                <target>Le modèle de article est cassé</target>
```
Comment (trickreich) [inline Resources/translations/sulu/backend.fr.xlf]: Is this translation correct?

File: Resources/scss/modules/list.scss [hunk 3]
```diff
@@ -9,3 +9,8 @@
         content: "\"";
     }
 }
+
+.husky-table tbody td .article-broken .cell-content {
+    color: $gray !important;
```
Comment (trickreich) [inline Resources/scss/modules/list.scss]: is `!important` really needed?

File: Resources/public/js/components/articles/list/main.js [hunk 4]
```diff
@@ -252,6 +251,42 @@ define([
             );
         },
 
+        actionCallback: function(id, article) {
```
Comment (trickreich) [inline Resources/public/js/components/articles/list/main.js]: Add JS doc

File: Resources/scss/modules/list.scss (hunk 3 above)
Comment (wachterjohannes) [reply Resources/scss/modules/list.scss]: yes else i would have to make the selector more complex ... copied from https://github.com/sulu/sulu/blob/develop/src/Sulu/Bundle/CategoryBundle/Resources/public/scss/modules/table-view.scss#L2

**Comment by wachterjohannes** [comment]
@trickreich fixed

File: Resources/translations/sulu/backend.nl.xlf [hunk 5]
```diff
@@ -194,6 +194,10 @@
                 <source>sulu_article.edit.page</source>
                 <target>Pagina</target>
             </trans-unit>
+            <trans-unit id="broken-template-01" resname="sulu_content.broken-template.title">
```
Comment (trickreich) [inline Resources/translations/sulu/backend.nl.xlf]: `sulu_content`?

File: Resources/translations/sulu/backend.fr.xlf [hunk 6]
```diff
@@ -194,6 +194,10 @@
                 <source>sulu_article.edit.page</source>
                 <target>Page</target>
             </trans-unit>
+            <trans-unit id="broken-template-01" resname="sulu_content.broken-template.title">
```
Comment (trickreich) [inline Resources/translations/sulu/backend.fr.xlf]: `sulu_content`?

File: Resources/translations/sulu/backend.en.xlf [hunk 7]
```diff
@@ -194,6 +194,10 @@
                 <source>sulu_article.edit.page</source>
                 <target>Page</target>
             </trans-unit>
+            <trans-unit id="broken-template-01" resname="sulu_content.broken-template.title">
```
Comment (trickreich) [inline Resources/translations/sulu/backend.en.xlf]: `sulu_content`?

File: Resources/translations/sulu/backend.de.xlf [hunk 8]
```diff
@@ -194,6 +194,10 @@
                 <source>sulu_article.edit.page</source>
                 <target>Seite</target>
             </trans-unit>
+            <trans-unit id="broken-template-01" resname="sulu_content.broken-template.title">
```
Comment (trickreich) [inline Resources/translations/sulu/backend.de.xlf]: `sulu_content`?

File: Resources/public/js/components/articles/list/main.js [hunk 9]
```diff
@@ -53,6 +61,10 @@ define([
             filterByTag: 'sulu_article.list.filter.by-tag',
             filterByPage: 'sulu_article.list.filter.by-page',
             filterByTimescale: 'sulu_article.list.filter.by-timescale',
+            brokenTemplateTitle: 'sulu_content.broken-template.title',
+            brokenTemplateMessage: 'sulu_article.broken-template.message',
```
Comment (trickreich) [inline Resources/public/js/components/articles/list/main.js]: `sulu_content`?

File: Resources/public/js/components/articles/list/main.js [hunk 10]
```diff
@@ -53,6 +61,10 @@ define([
             filterByTag: 'sulu_article.list.filter.by-tag',
             filterByPage: 'sulu_article.list.filter.by-page',
             filterByTimescale: 'sulu_article.list.filter.by-timescale',
+            brokenTemplateTitle: 'sulu_content.broken-template.title',
```
Comment (trickreich) [inline Resources/public/js/components/articles/list/main.js]: `sulu_article`?

**Pull request merged**